app/assets/stylesheets/application.scss

```diff
@@ -47,6 +47,18 @@
    }
  }
 
+ // .card-photo-alignment {
+ //   display: flex;
```

Comment:
Make sure to take out any comments if you're not planning on using them in this PR